### PR TITLE
Now we have working routes. When users log in as Tenants, they are na…

### PR DESCRIPTION
**Pull request by ItisEeron**
I have updated the files to now include a function that takes care of navigation between Tenant and Landlord accounts. This function can be found in the /src/Routes/Routes.tsx file. I also was required to update the fetchAccount and generateAccount actions in the /accounts/actions.tsx file. I also needed to update all the components in the SignUp, RoopairsLogIn, and Login directories. These should work but they have not been unit tested.

### CODE REVIEW

**Comment by YellowRainBoots** [comment]
this doesn't look right? under propertyStackConfig_PM u changed it to TenantProperties

Screen Shot 2020-01-22 at 9 50 13 PM

File: homepairs/homepairsUI/src/Routes/Routes.tsx [hunk 1]
```diff
@@ -75,7 +76,7 @@ const innerStackConfig: any = {
 
 //these should be separated into different files for each Route (AccountProperties, Service Request, Account)
 const propertyStackConfig_PM = {
-  initialRouteName: 'AccountProperties',
+  initialRouteName: 'TenantProperties',
```
Comment (YellowRainBoots) [inline homepairs/homepairsUI/src/Routes/Routes.tsx]: why is PM config initially routing to tenant properties?

**Comment by YellowRainBoots** [comment]
I'll wait to delete the branch until I've got your approval + we've had a chance to work with it, since we don't have full testing in place yet.

File: homepairs/homepairsUI/src/Routes/Routes.tsx (hunk 1 above)
Comment (ItisEeron) [reply homepairs/homepairsUI/src/Routes/Routes.tsx]: I outright removed the propertyStack_PM and propertyStack_TENANT and simply put them in the same stack. I resolve them with a new function called ChooseMainPage() which all the fetchAccount actions call. We just default to TenantProperties page. However, this initial Route should never be called if we are making sure to always calling new function I defined. 

We should rename the configuration however. I can update that right now.

**Pull request merged**